src/components/pages/order/Main/Admin/AdminPanels.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from "react";
+import styled from "styled-components";
+import TabAdd from "./tabs/TabAdd";
+import TabUpdate from "./tabs/TabUpdate";
+import OrderContext from "../../../../../context/OrderContext";
+
+function AdminPanels() {
+  const { isAddSelected, isEditSelected } = useContext(OrderContext);
+  return (
+    <AdminPanelStyle>
+      {isAddSelected ? <TabAdd /> : ""}
```

Comment:
```suggestion
      {isAddSelected && <TabAdd />}
```

pareil pour la ligne du dessous